archetypes/data-playground.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "{{ replaceRE "^[-0-9]{10}-" "" .Name | replaceRE "-" " " | title }}"
+slug: "{{ replaceRE "^[-0-9]{10}-" "" .Name }}"
+draft: true
+image: img/data-playground/
+description: 'Add Description Here'
+license: "Add License Here"
+producer: "Add Producer Here"
+api_key: "Add Api key Here"
+api_type: "Add Api type Here"
+sdks: " Add sdk list seperate with comma"
+limits: "Add limits here"
+data: " Add data here"
+account_required: " Add required account"
+
+---
+
+Add data products showcase 
```

Comment:
Should this be commented out?

Reply:
it has been added to give instructions to the users who wants to add a new playground

Reply:
Don't worry about this for now; the users can add any content here, including a table.